packages/eslint-plugin/__tests__/stylex-sort-keys-test.js

```diff
@@ -286,5 +358,319 @@ eslintTester.run('stylex-sort-keys', rule.default, {
+      ],
+    },
+    {
+      code: `
+      import stylex from 'stylex';
+      const styles = stylex.create({
+        foo: {
+          /*
+          *
+          * foo
+          * bar
+          * baz
+          *
+          */
+          backgroundColor: 'red',
+          alignItems: 'center'
+        }
+      })
+      `,
+      output: `
+      import stylex from 'stylex';
+      const styles = stylex.create({
+        foo: {
+          alignItems: 'center',
+          /*
+          *
+          * foo
+          * bar
+          * baz
+          *
+          */
+          backgroundColor: 'red',
+        }
+      })
+      `,
+      errors: [
+        {
+          message:
+            'StyleX property key "alignItems" should be above "backgroundColor"',
+        },
+      ],
+    },
+    {
+      code: `
+      import stylex from 'stylex';
+      const styles = stylex.create({
+        foo: {      
+          backgroundColor: 'red',             //       foo
+          alignItems: 'center'       // baz
+        }
+      })
+      `,
+      output: `
+      import stylex from 'stylex';
+      const styles = stylex.create({
+        foo: {      
+          alignItems: 'center',       // baz
+          backgroundColor: 'red',             //       foo
+        }
+      })
+      `,
+      errors: [
+        {
+          message:
+            'StyleX property key "alignItems" should be above "backgroundColor"',
+        },
+      ],
+    },
+    {
+      code: `
+      import stylex from 'stylex';
+      const styles = stylex.create({
+        foo: {
+          /*
+          * foo
+          */ backgroundColor: 'red',
+          alignItems: 'center'
+        }
+      })
+      `,
+      output: `
+      import stylex from 'stylex';
+      const styles = stylex.create({
+        foo: {
+          alignItems: 'center',
+          /*
+          * foo
+          */ backgroundColor: 'red',
+        }
+      })
+      `,
+      errors: [
+        {
+          message:
+            'StyleX property key "alignItems" should be above "backgroundColor"',
+        },
+      ],
+    },
+    {
+      code: `
+      import stylex from 'stylex';
+      const styles = stylex.create({
+        foo: {
+          // foo
+
+          backgroundColor: 'red',
+          alignItems: 'center'
+        }
+      })
+      `,
+      output: `
+      import stylex from 'stylex';
+      const styles = stylex.create({
+        foo: {
+          // foo
+
+          alignItems: 'center',
+          backgroundColor: 'red',
+        }
+      })
+      `,
+      errors: [
+        {
+          message:
+            'StyleX property key "alignItems" should be above "backgroundColor"',
+        },
+      ],
+    },
+    {
+      options: [{ allowLineSeparatedGroups: true }],
+      code: `
+      import { create as cr } from '@stylexjs/stylex';
+      const styles = cr({
+        button: {
+          alignItems: 'center',
+          display: 'flex',
+          // foo
+
```

Comment:
found another edge case that was not handled correctly imo.

if there is at least one empty line between the comment and the node, that comment is not considered to be part of the property context